rpcs/ledgerService.go

```diff
@@ -110,19 +83,19 @@ func (ls *LedgerService) ServeHTTP(response http.ResponseWriter, request *http.R
 	genesisID, hasGenesisID := pathVars["genesisID"]
 	if hasVersionStr {
 		if versionStr != "1" {
-			logging.Base().Debug("http block bad version", versionStr)
+			logging.Base().Debug("http ledger bad version", versionStr)
 			response.WriteHeader(http.StatusBadRequest)
```

Comment:
The StatusBadRequest and other failure codes do not provide much information. 
Can we return the error message with the response to help the client identify the issue?
We can we use something like ErrorResponse.

Reply:
The header need to remain StatusBadRequest; however, we can return whatever information we want in the response body.

Reply:
That will be very useful.